x-team/auto-ui
# Add auto-size feature to textarea fields

**Release Type:** *New feature* 


Fixes x-team/auto#1255

## Description

- Adds an auto-size feature to textarea fields.

## Checklist



## Related PRs

branch | PR
------ | ------
`1255-textareas-auto-grow-to-fit-content` | [Auto](https://github.com/x-team/auto/pull/1265)

## Steps to Test or Reproduce

- Navigate to the `TextareaEditor` component section.
  - Make sure the descriptions match the behaviors.
- Navigate to the `InputField ` component section.
  - Make sure the descriptions match the behaviors.
